src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -139,28 +138,23 @@ public String key() {
 
   /*
    * PlaybackDelegate implements Adobe's MediaHeartbeatDelegate interface. This implementation allows us
-   * to define logic that returns the position of a video playhead during a video session.
+   * to write logic and returns the position of a video playhead during a video session.
    */
   static class PlaybackDelegate implements MediaHeartbeatDelegate {
     /** The system time at which the current instance of PlaybackDelegate was instantiated * */
     final long initialTime;
-    /**
-     * The current playhead position; atomic in case this value is accessed by multiple threads *
-     */
-    AtomicLong playheadPosition = new AtomicLong();
-    /**
-     * The position of the playhead when the video is paused; atomic in case this value is accessed
-     * by multiple threads *
-     */
-    AtomicLong pausedPlayheadPosition = new AtomicLong();
+    /** The current playhead position * */
```

Comment:
why is there an extra `*` here? should be `/** The current playhead position */`

Reply:
(same for the other comments)